### PR TITLE
Add Collaboration Tree (Demo)

### PR DESCRIPTION
**Pull request by maria-le**
https://github.com/F4IF/ctree-demo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # PolymerProjects
 
-Projects using Polymer.
+### Projects using Polymer
```
Comment (abdonrd) [inline README.md]: Please, split this in another PR.

File: README.md [hunk 2]
```diff
@@ -54,6 +58,7 @@ Sorted alphabetically
 | [:link:](https://www.catminingexperience.com) | Caterpillar - MINExpo 2016 | `1.6` | | |
 | [:link:](https://www.cerberusapp.com) | Cerberus | `1.8` | | |
 | [:link:](https://www.cloudstitch.com) | Cloudstitch | `1.8` | | |
+| [:link:](https://github.com/F4IF/ctree-demo) | Collaboration Tree (Demo) | `1.7` | :heavy_check_mark: | [Source](https://github.com/F4IF/ctree-demo)  |
```
Comment (abdonrd) [inline README.md]: The first URL should be the online demo. Can you update it?

**Review by abdonrd — CHANGES_REQUESTED** [review]
Thanks for the PR! 👏

File: README.md (hunk 2 above)
Comment (maria-le) [reply README.md]: I double and triple checked, but I don't think there is any online demo. The user has to clone the repo and serve locally AFAIK.

Comment (abdonrd) [reply README.md]: Then update with a simple `-`, like some Chrome projects. Thanks!

**Review by abdonrd — APPROVED** [review]
Thanks @maria-le!
